### PR TITLE
Adds tests to show all fields that are filled out, show up

### LINKED ISSUE #133: Item View Fields

### PR DESCRIPTION
**Pull request by straleyb**
Closes #133 Already configured. Extended tests to show empty fields don't get displayed and all other filled out fields get dispalyed.

### CODE REVIEW

File: spec/views/catalog/_show_default.html.erb_spec.rb [hunk 1]
```diff
@@ -24,4 +27,14 @@
...
   end
+  it "should show all other fields" do
+    expect(rendered).to have_content("Date")
+    expect(rendered).to have_content("06/17/2015")
+    expect(rendered).to have_content("Creator")
+    expect(rendered).to have_content("Brandon")
+  end
+  it "should not show blank fields" do
+    expect(rendered).to_not have_content("Material")
+    expect(rendered).to_not have_content("Space Smores")
```
Comment (terrellt) [inline spec/views/catalog/_show_default.html.erb_spec.rb]: When would this ever show up?

Comment (straleyb) [inline spec/views/catalog/_show_default.html.erb_spec.rb]: Ah so is it enough that I checked that material didn't show up on the page?

Comment (terrellt) [inline spec/views/catalog/_show_default.html.erb_spec.rb]: You never added Space Smores anywhere right? It showing up would be pretty crazy

Comment (straleyb) [inline spec/views/catalog/_show_default.html.erb_spec.rb]: :+1:

**Pull request merged**